### DISCUSSION TITLE
msg="Target URL https://app.test.com/ is not under the protected domain authelia.test.com"

### DISCUSSION THREAD

**Discussion by shankark1991** [authelia/authelia#6224] [Q&A] [answered]
time="2023-11-04T16:54:21Z" level=trace msg="Headers=GET /api/verify HTTP/1.1\r\nUser-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.61\r\nHost: x.x.x.x:9091\r\nX-Original-Url: https://app.test.com/\r\nX-Original-Method: GET\r\nX-Forwarded-Method: GET\r\nX-Forwarded-Proto: https\r\nX-Forwarded-Host: app.test.com\r\nX-Forwarded-Uri: /\r\nX-Forwarded-For: x.x.x.x\r\nSec-Ch-Ua: \"Chromium\";v=\"118\", \"Microsoft Edge\";v=\"118\", \"Not=A?Brand\";v=\"99\"\r\nSec-Ch-Ua-Mobile: ?0\r\nSec-Ch-Ua-Platform: \"Windows\"\r\nUpgrade-Insecure-Requests: 1\r\nAccept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7\r\nSec-Fetch-Site: none\r\nSec-Fetch-Mode: navigate\r\nSec-Fetch-User: ?1\r\nSec-Fetch-Dest: document\r\nAccept-Encoding: gzip, deflate, br\r\nAccept-Language: en-US,en;q=0.9\r\n\r\n" method=GET path=/api/verify remote_ip=x.x.x.x
time="2023-11-04T16:54:21Z" level=trace msg="Using X-Original-URL header content as targeted site URL" method=GET path=/api/verify remote_ip=x.x.x.x
time="2023-11-04T16:54:21Z" level=error msg="Target URL https://app.test.com/ is not under the protected domain authelia.test.com" method=GET path=/api/verify remote_ip=x.x.x.x stack="github.com/authelia/authelia/v4/internal/handlers/handler_verify.go:459 VerifyGET.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/bridge.go:54       (*BridgeBuilder).Build.func1.1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:16      SecurityHeaders.func1\ngithub.com/fasthttp/router@v1.4.14/router.go:427                        (*Router).Handler\ngithub.com/valyala/fasthttp@v1.43.0/http.go:154                         (*Response).StatusCode\ngithub.com/valyala/fasthttp@v1.43.0/server.go:2338                      (*Server).serveConn\ngithub.com/valyala/fasthttp@v1.43.0/workerpool.go:224                   (*workerPool).workerFunc\ngithub.com/valyala/fasthttp@v1.43.0/workerpool.go:196                   (*workerPool).getCh.func1\nruntime/asm_amd64.s:1594                                                goexit"
![redirection](https://github.com/authelia/authelia/assets/133733148/b93525c7-9964-40c2-9dd1-1afa2b2e2a75)

**Comment by nightah** [comment, marked as answer]
Your session domain is set to `authelia.test.com` which cannot set cookies for `app.test.com` change the configuration to `test.com` instead.

**Reply by shankark1991** [reply]
Hi Sir,

Thanks you so much, It's worked. But after redirection, it's trying to GET the application. 
[authelia_logs1.txt](https://github.com/authelia/authelia/files/13266112/authelia_logs1.txt)

Please find the attachment and advise me to solve this issue.

**Reply by shankark1991** [reply]
Hi Sir/Mam,

Kindly help me to solve this issue.

**Reply by james-d-elliott** [reply]
What is the issue you're experiencing? The logs look fine to me.

**Reply by shankark1991** [reply]
Hi James,

Actually I'm trying to redirect from authelia to my application (with or without authentication) with MFA feature. The application I'm using is PWM (https://github.com/pwm-project/pwm).

Please help me.

**Reply by james-d-elliott** [reply]
You can either visit the protected application prior to authenticating, or set the default redirection url if you'd like to redirect there by default. If visiting the application then authenticating does not redirect you either the proxy is not configured per our docs or there will be a log message indicating why.
